src/chatgptClient.ts

```diff
@@ -23,6 +23,16 @@ export async function getSuggestions(
 
   // extract the json from the response
   const result = response.data.choices[0].text ?? ''
```

Comment:
### Line 25
## CodeGuard Suggestions
**Suggestion:** Use the 'result' variable and the JS indexOf & substring methods to extract the json
**Reason:** This a cleaner and more efficient way to get the desired value